tests/integration/sdk_api/alerts/test_alerts.py

```diff
@@ -674,6 +732,45 @@ def _validate_alert(
         if alert_count:
             assert alert.count == alert_count
 
+    @staticmethod
+    def _validate_alert_activation(
+        alert_activation,
+        project_name=None,
+        alert_name=None,
+        alert_event_name=None,
+        alert_severity=None,
+        alert_criteria=None,
+        alert_entity_id=None,
+        alert_entity_kind=None,
+        alert_event_kind=None,
+        number_of_events=None,
+        alert_notifications=None,
```

Comment:
in your validations you only pass to this method: alert_activation, project_name, alert_name & number of events, 
so the rest of the params here are redundant.
also, this seems like a code duplication of the _validate_alert method here in the same test, so why not combine them and have one validator ? ( either pass it the alert_data or the alert_activation )

Reply:
@Yacouby I think it's good to have them separated as even though some fields do match, some of them don't, so it will decrease readability, I'd prefer it as is.